src/tv2-common/blueprintConfig.ts

```diff
@@ -39,10 +39,8 @@ export interface TableConfigSchema {
 
 export interface TableConfigGraphicsSetup {
 	Name: string
-	VcpConcept: string
-	OvlShowId: string
-	FullShowId: string
-	DveLayoutFolder: string
+	OvlShowName: string
+	FullShowName?: string
```

Comment:
What is the reason for this being optional? 
Isn't this kinda the same as the `iNewsColumnStyle` from Rasmus PR?

Reply:
This is due to how the types and Viz graphics are structured - they currently are in tv2-common and we still support Viz graphics in both Show Styles, but only the Gallery blueprints expose the `FullShowName`. We spoke about removing that cross-support and sticking with Viz in Gallery (AFVD) Show Style and HTML in Qbox (Offtube) Show Style, but if does not seem to be a part of this task.